cmd/operator/main.go

```diff
@@ -158,6 +166,13 @@ func main() {
 		os.Exit(1)
 	}
 
+	var tokenCache *cache.TokenCache
+	if tokenCacheMaxSize > 0 {
+		tokenCache = cache.NewTokenCache(tokenCacheMaxSize,
+			cache.WithMaxDuration(tokenCacheMaxDuration),
+			cache.WithMetricsRegisterer(reporter.Registerer()))
```

Comment:
What's the metric name and labels, I can't tell from this config, but it needs to be `flux_` prefixed.

Reply:
Maybe we should add this `flux_` prefix in `fluxcd/pkg/cache`? See this PR: https://github.com/fluxcd/pkg/pull/877

https://github.com/fluxcd/pkg/pull/877/files#diff-1eb58a7a2b6b183a0788cff2b8e29b60c7be6afff5bb2e07068b842a8744ddd6R73

In this PR I'm hard-coding `token_` internally in the `cache.TokenCache` constructor since it's already a "terminal" use case, we won't have specialized token caches, only a single token cache. Maybe this prefix should be `flux_cache_`?

(Please stamp this 877 and https://github.com/fluxcd/pkg/pull/878)

Reply:
Names and labels are here: https://github.com/fluxcd/pkg/blob/ff04927148b1b1407f07f58ffc3bcded59eb7a14/cache/metrics.go#L46

Reply:
We cant' hardcode the metric name in the package, Flux Operator needs `flux_` while the Flux Controller would use the GitOps Toolkit prefix `gotk_`.

Please post here the full metric name and labels, this needs to be documented in: https://fluxcd.control-plane.io/operator/monitoring/

Reply:
Cool I'll remove this from 877. Can you please stamp it anyway (and also 878)? I think it'd be nice to have a `token-cache-max-duration` flag as well. (For OIDC tokens for example users may want to decrease this max duration as the token caches an impersonation permission that could have been revoked).

Reply:
I added the metrics to the PR description, thanks!